api/src/paths/critter-data/critters/index.test.ts

```diff
@@ -0,0 +1,63 @@
+import Ajv from 'ajv';
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { CritterbaseService, IBulkCreate } from '../../../services/critterbase-service';
+import { getRequestHandlerMocks } from '../../../__mocks__/db';
+import * as createCritter from './index';
+
+chai.use(sinonChai);
+
+describe('paths/critter-data/critters/post', () => {
+  const ajv = new Ajv();
+
+  it('is valid openapi v3 schema', () => {
+    expect(ajv.validateSchema((createCritter.POST.apiDoc as unknown) as object)).to.be.true;
+  });
+
+  const payload: IBulkCreate = {
+    critters: [],
+    captures: [],
+    mortalities: [],
+    locations: [],
+    markings: [],
+    qualitative_measurements: [],
+    quantitative_measurements: [],
+    families: [],
+    collections: []
+  };
+
+  describe('createCritter', () => {
+    afterEach(() => {
+      sinon.restore();
+    });
+    it('should succeed', async () => {
+      const mockCreateCritter = sinon.stub(CritterbaseService.prototype, 'createCritter').resolves({ count: 0 });
+      const { mockReq, mockRes, mockNext } = getRequestHandlerMocks();
+      mockReq.body = payload;
+      const requestHandler = createCritter.createCritter();
+
+      await requestHandler(mockReq, mockRes, mockNext);
+
+      expect(mockCreateCritter.calledOnce).to.be.true;
+      expect(mockCreateCritter).calledWith(payload);
+      expect(mockRes.statusValue).to.equal(200);
+      expect(mockRes.json.calledWith({ count: 0 })).to.be.true;
+    });
+    it('should fail', async () => {
+      const mockError = new Error('mock error');
+      const mockCreateCritter = sinon.stub(CritterbaseService.prototype, 'createCritter').rejects(mockError);
+
+      const { mockReq, mockRes, mockNext } = getRequestHandlerMocks();
+      const requestHandler = createCritter.createCritter();
+
+      try {
+        await requestHandler(mockReq, mockRes, mockNext);
+        expect.fail();
+      } catch (actualError) {
+        expect(actualError).to.equal(mockError);
+        expect(mockCreateCritter.calledOnce).to.be.true;
```

Comment:
nit: in general, a better way to write this assertion is:

`expect(mockAddUser).to.have.been.calledOnce;`

and even better yet (if there are params):

`expect(mockAddUser).to.have.been.calledOnceWith(<params>);`